example/swagger-files/petstore.json

```diff
@@ -1,827 +0,0 @@
-{
```

Comment:
Why it is deleted?

Reply:
petstore use "swagger: 2.0" and we use "openapi". Petstore causes error and, since we have to support just openapi, we removed petstore and added instead a "openapi 3" example.
Petstore was not a useful example for what we need

Reply:
Ok perfect. Could we write in the README that we support only OAS3?

Reply:
mmmh wait what? This file is currently working, if it does not work anymore thare's been a breaking change

Reply:
Yes but we support only OAS3, so it's not a big deal to support swagger 2.
But I checked the [example](https://github.com/mia-platform/api-explorer/blob/master/example/src/SpecFetcher.jsx) and see that the examples are converted to OAS3. So, I suppose that a swagger 2.0 should be converted to a oas3. If this does not works, it's a prbleem

Reply:
I'll check

Reply:
Yes, what I meant is that although the library supports OAS3 it can convert swagger2 files so unless we remove the conversion we should keep these files.

Reply:
Fixed. We succeeded to support petstore again.
It's was due to reference that was not resolved.

Reply:
Great!